lib/b2b/sso.ts

```diff
@@ -32,93 +30,207 @@ export interface SAMLConnection {
   organization_id: string;
   connection_id: string;
   status: string;
-  attribute_mapping: Record<string, string>;
   idp_entity_id: string;
   display_name: string;
   idp_sso_url: string;
   acs_url: string;
   audience_uri: string;
   signing_certificates: X509Certificate[];
   verification_certificates: X509Certificate[];
+  attribute_mapping?: Record<string, any>; // eslint-disable-line @typescript-eslint/no-explicit-any
 }
 
-export interface B2BSSOGetConnectionsRequest {
-  organization_id: string;
-}
-
-export interface B2BSSOGetConnectionsResponse extends BaseResponse {
-  saml_connections: SAMLConnection[];
-  oidc_connections: OIDCConnection[];
-}
-
-export interface B2BSSODeleteConnectionRequest {
-  organization_id: string;
-  connection_id: string;
-}
-
-export interface B2BSSODeleteConnectionResponse extends BaseResponse {
-  connection_id: string;
+export interface X509Certificate {
+  certificate_id: string;
+  certificate: string;
+  issuer: string;
+  created_at?: Date;
+  expires_at?: Date;
```

Comment:
This type was `string` before - are we standardizing on using the `Date` type for times?